drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/HtmlUnitDriverFactory.java

```diff
@@ -86,4 +101,22 @@ public Capabilities getCapabilities(WebDriverConfiguration configuration, boolea
     protected String getDriverReadableName() {
         return BROWSER_CAPABILITIES;
     }
+
+    private void setClientOptions(WebClient webClient, String htmlUnitWebClientOptions) {
+        final WebClientOptions webClientOptions = webClient.getOptions();
+
+        Map<String, String> clientOptions = new LinkedHashMap<>();
+        final String[] options = htmlUnitWebClientOptions.split(",");
+        for (String option : options) {
+            final String[] keyValue = option.split("=");
+            if (keyValue.length == 2) {
+                String key = keyValue[0].trim();
```

Comment:
Remember that the whole property can be a multiline string - use `StringUtils.trimMultiline` before you split it for the first time